benchmarktests/target_auth_gcp.go

```diff
@@ -95,18 +91,20 @@ type GCPTestRoleConfig struct {
 }
 
 func (g *GCPAuth) ParseConfig(body hcl.Body) error {
-	g.config = &GCPTestConfig{
-		Config: &GCPAuthTestConfig{
-			GCPAuthConfig:     &GCPAuthConfig{},
-			GCPTestRoleConfig: &GCPTestRoleConfig{},
-		},
+	testConfig := &struct {
+		Config *GCPAuthTestConfig `hcl:"config,block"`
+	}{Config: &GCPAuthTestConfig{
+		GCPAuthConfig:     &GCPAuthConfig{},
+		GCPTestRoleConfig: &GCPTestRoleConfig{},
+	},
```

Comment:
Is there a reason for the new variable here instead of storing this in `g.config` like it previously was?

Reply:
I think we would lose the hcl struct tags (``hcl:"config,block"`` if we did that. 
it would look something like 
```
g.config = &GCPAuthTestConfig{
		GCPAuthConfig:     &GCPAuthConfig{},
		GCPTestRoleConfig: &GCPTestRoleConfig{},
	}
```